statsmodels/discrete/tests/test_conditional.py

```diff
@@ -293,12 +292,12 @@ def test_conditional_mlogit1():
                     [0.65190315, 0.59653771]]),
         rtol=1e-5, atol=1e-5)
 
-def test_conditional_mlogit2():
+def test_conditional_mnlogit2():
```

Comment:
can this be given a more informative name?  e.g. "test_conditional_mnlogit_highly_collinear" or something